### DISCUSSION TITLE
Overlay Editor thinks its been modified

### DISCUSSION THREAD

**Discussion by ssagerian** [AllskyTeam/allsky-modules#59] [General]
Every morning, I have to go to the module manager and re enable overlays for daytime. In addition, going to the overlay editor, it thinks that something has been modified (without actually modifying the overlay).

**Comment by Alex-developer** [comment]
Can you try turning off the watchdog in the modules manager settings

**Comment by ssagerian** [comment]
Ok, turned off the watch dog, will post tomorrow..

**Comment by ssagerian** [comment]
Turning off the watch dog does fix the issue of the overlay module not being enabled.
I need to recheck another morning to see if the overlay module manager thinks its been modified, this may be a separate issue.

**Reply by Alex-developer** [reply]
Thats interesting an would suggest that the overlay is taking too long to process. What pi is this running on?

**Reply by ssagerian** [reply]
This setup is using a Pi 3 Model B Revision 1.2 (1 GB).
Do we have a lower limit restriction on the Pi model?

**Reply by Alex-developer** [reply]
I suspect its probably the memory limit on the Pi3. I'll setup a Pi3 with that release this week and give it a test

**Reply by ssagerian** [reply]
![Screenshot 2024-01-07 at 15-43-41 System - WebUI](https://github.com/AllskyTeam/allsky-modules/assets/3128943/75cdc9c4-fadc-4da7-87f4-5ab8b0155ca5)
I havent taken notice of the memory utilized,  I have seen, in the morning at night to day transitions the cpu utilization can go up to 99%

**Reply by ssagerian** [reply]
rather its CPU Load..

**Comment by Alex-developer** [comment]
I have found and fixed the overlay thinking its been modified. It will be out in the next release so closing this.
